### PR TITLE
[DOCFIX] update namespace management doc

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

File: docs/en/advanced/Namespace-Management.md [hunk 1]
```diff
@@ -9,28 +9,41 @@ priority: 0
...
+Alluxio enables effective data management across different storage systems through its use of transparent naming and mounting API.
 
-Intro: Introduce concepts such as:
-- Alluxio's unified namespace
-- UFS namespace
-- transparent naming
+## Unified namespace
+One of the key benefit that Alluxio provides is a unified namespace to the applications.
+This is an abstraction that allows applications to access multiple independent storage systems through the same name space and interface.
+Applications simply communicates with Alluxio and Alluxio manages communications with the different underlying storage system.
```
[the remark this answers is not in the document]
Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: fixed

File: docs/en/advanced/Namespace-Management.md [hunk 2]
```diff
@@ -9,28 +9,41 @@ priority: 0
...
-  include example
-## Nested Mount Points
-  include example
-# Ufs Metadata Sync
-  this section is probably the contents from Loading-Under-File-Storage-Metadata
-# Examples
-  reuse existing examples
+![unified]({{ site.baseurl }}/img/screenshot_unified.png)
+
+By default, Alluxio namespace is mounted onto the directory specified by the
```
[the remark this answers is not in the document]
Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: fixed

File: docs/en/advanced/Namespace-Management.md [hunk 3]
```diff
@@ -53,32 +66,127 @@ metadata will be loaded into Alluxio the first time they are accessed (e.g. when
...
-Alluxio provides a mounting API that makes it possible to use Alluxio to access data across multiple
-data sources.
+# Mounting Under File System (UFS)
+Mounting in Alluxio works similarly to mounting a volume in a Linux file system.
+The mount command attaches a UFS to the file system tree in the Alluxio namespace. 
+The command line interface (CLI) for mounting has the following syntax.
 
-![unified]({{ site.baseurl }}/img/screenshot_unified.png)
+```bash
+$ bin/alluxio fs mount /path/in/alluxio/namespace /path/to/ufs
```
[the remark this answers is not in the document]
Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: done

File: docs/en/advanced/Namespace-Management.md [hunk 4]
```diff
@@ -53,32 +66,127 @@ metadata will be loaded into Alluxio the first time they are accessed (e.g. when
...
+```bash
+$ bin/alluxio fs mount /path/in/alluxio/namespace /path/to/ufs 
+```
 
-By default, Alluxio namespace is mounted onto the directory specified by the
-`alluxio.underfs.address` property of Alluxio configuration; this directory identifies the
-"primary storage" for Alluxio. In addition, users can use the mounting API to add new and remove
-existing data sources:
+## Root Mount Point
+The root mount point of the Alluxio namespace can be specified by adding the following line to the configuration file in `conf/alluxio-site.properties`.
```
[the remark this answers is not in the document]
Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: done

File: docs/en/advanced/Namespace-Management.md [hunk 5]
```diff
@@ -53,32 +66,127 @@ metadata will be loaded into Alluxio the first time they are accessed (e.g. when
...
 
-For example, the primary storage could be HDFS and contain user directories; the `Data` directory
-might be stored in an S3 bucket, which is mounted to the Alluxio namespace through
-```java
-mount(new AlluxioURI("alluxio://host:port/Data"), new AlluxioURI("s3a://bucket/directory"));
+## Nested Mount Points
+In addition to the root mount point, other under filesystems can be mounted into Alluxio namespace by using the mount command. 
+
+```bash
+$ bin/alluxio fs mount /path/in/alluxio/namespace /path/to/ufs
```
[the remark this answers is not in the document]
Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: done

File: docs/en/advanced/Namespace-Management.md [hunk 6]
```diff
@@ -53,32 +66,127 @@ metadata will be loaded into Alluxio the first time they are accessed (e.g. when
...
+
+```bash
+$ bin/alluxio fs mount /path/in/alluxio/namespace /path/to/ufs 
 ```
 
-## Examples
+The result of this mount command is that the ufs path will be attached to the Alluxio namespace at the designated location.
+Note that mount points can be nested as well. For example, we can mount one UFS at  `alluxio:///path1`, and then mount another UFS at  `alluxio:///path1/path2`
+
+# UFS Metadata Sync
```
[the remark this answers is not in the document]
Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: done

**Comment by yuzhu** [comment]
@gpang comments addressed, PTAL

File: docs/en/advanced/Namespace-Management.md [hunk 7]
```diff
@@ -53,32 +66,133 @@ metadata will be loaded into Alluxio the first time they are accessed (e.g. when
...
+```
+alluxio.underfs.address=hdfs://HDFS_HOSTNAME:9000
+``` 
 
-```java
-void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath);
-void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountOptions options);
-void unmount(AlluxioURI path);
-void unmount(AlluxioURI path, UnmountOptions options);
+If the root mount point requires additional parameters such as credentials, it can be specified in the configuration file as well.
```
Comment (gpang) [inline docs/en/advanced/Namespace-Management.md]: For the mount options for the root mount point, they should use the configuration prefix:

`alluxio.master.mount.table.root.option.<some alluxio property>`

in the file.

Example:

```
alluxio.master.mount.table.root.option.alluxio.security.underfs.hdfs.kerberos.client.principal=client
alluxio.master.mount.table.root.option.alluxio.security.underfs.hdfs.kerberos.client.keytab.file=keytab
alluxio.master.mount.table.root.option.alluxio.security.underfs.hdfs.impersonation.enabled=true
alluxio.master.mount.table.root.option.alluxio.underfs.hdfs.version=hdp-2.6
```

Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: configuration options added.

**Comment by yuzhu** [comment]
@gpang PTAL.

**Comment by gpang** [comment]
Jenkins, test this please

File: docs/en/advanced/Namespace-Management.md [hunk 8]
```diff
@@ -53,32 +66,148 @@ metadata will be loaded into Alluxio the first time they are accessed (e.g. when
...
+
+If the root mount point requires additional parameters such as aws credentials, it can be specified in the configuration file as well. 
 
-```java
-void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath);
-void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountOptions options);
-void unmount(AlluxioURI path);
-void unmount(AlluxioURI path, UnmountOptions options);
+```
+aws.accessKeyId=<AWS_ACCESS_KEY_ID>
```
Comment (gpang) [inline docs/en/advanced/Namespace-Management.md]: these should also be prefixed with `alluxio.master.mount.table.root.option.`, right?

Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: fixed

**Pull request merged**